src/vivarium_public_health/risks/data_transformations.py

```diff
@@ -228,16 +231,18 @@ def get_relative_risk_data(builder, risk: EntityString, target: TargetString):
 
     else:
         relative_risk_data = relative_risk_data.drop(columns=["parameter"])
+        value_cols = ["value"]
 
-    return relative_risk_data
+    return relative_risk_data, value_cols
 
 
 def load_relative_risk_data(
-    builder, risk: EntityString, target: TargetString, source_type: str
-):
-    relative_risk_source = builder.configuration[f"effect_of_{risk.name}_on_{target.name}"][
-        target.measure
-    ]
+    builder: Builder, risk: EntityString, target: TargetString, source_type: str
+) -> pd.DataFrame:
+    from vivarium_public_health.risks import RiskEffect
```

Comment:
Is there a reason why you're doing this import here and on ln 470 inline and not at the top of the file?